### PR TITLE
JAMES-2593 Package Swift product

### PR DESCRIPTION
**Pull request by trantienduchn**

### CODE REVIEW

**Comment by trantienduchn** [comment]
```
[420d188f50f087821809bc62887d4a7b6a1b5b1f] [ERROR] Errors: 
[420d188f50f087821809bc62887d4a7b6a1b5b1f] [ERROR]   SpamAssassinInvokerTest.learnAsHamShouldAllowToForgetSpam » ContainerFetch Can.......
```
test this please

**Comment by trantienduchn** [comment]
```
[420d188f50f087821809bc62887d4a7b6a1b5b1f] [ERROR] Errors: 
[420d188f50f087821809bc62887d4a7b6a1b5b1f] [ERROR]   SpamAssassinInvokerTest.learnAsHamShouldAllowToForgetSpam » ContainerFetch Can.......
```
test this please

File: src/site/markdown/server/install/guice-cassandra-rabbitmq-swift.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
-# Guice-Cassandra-Rabbitmq installation guide
+# Guice-Cassandra-Rabbitmq-Swift installation guide
 
 ## Building
 
 ### Requirements
 
  - Java 8 SDK
- - Docker ∕ ElasticSearch 2.4.6 and Cassandra 3.11.3
+ - Docker ∕ ElasticSearch 2.4.6, RabbitMQ Management 3.3.7, Swift ObjectStorage and Cassandra 3.11.3
```
Comment (mbaechler) [inline src/site/markdown/server/install/guice-cassandra-rabbitmq-swift.md]: which version of Swift ?

File: src/site/markdown/server/install/guice-cassandra-rabbitmq-swift.md [hunk 2]
```diff
@@ -28,6 +28,7 @@ mvn clean install
  * Cassandra 3.11.3
  * ElasticSearch 2.4.6
  * RabbitMQ-Management 3.7.7
+ * Swift ObjectStorage
```
Comment (mbaechler) [inline src/site/markdown/server/install/guice-cassandra-rabbitmq-swift.md]: version ?

**Comment by trantienduchn** [comment]
```
[420d188f50f087821809bc62887d4a7b6a1b5b1f] [ERROR] Errors: 
[420d188f50f087821809bc62887d4a7b6a1b5b1f] [ERROR]   SpamAssassinInvokerTest.learnAsHamShouldAllowToForgetSpam » ContainerFetch Can.......
```
test this please

File: src/site/markdown/server/install/guice-cassandra-rabbitmq-swift.md (hunk 1 above)
Comment (mbaechler) [reply src/site/markdown/server/install/guice-cassandra-rabbitmq-swift.md]: see https://github.com/jeantil/openstack-swift-keystone-docker , it should be Swift 2.15.1

Comment (trantienduchn) [reply src/site/markdown/server/install/guice-cassandra-rabbitmq-swift.md]: oh, cool!!! I was struggling with finding its version by doing docker inspect =.=. Thank you any way!

**Comment by trantienduchn** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

File: dockerfiles/run/docker-compose.yml [hunk 3]
```diff
@@ -34,4 +35,10 @@ services:
     image: rabbitmq:3.7.7-management
     ports:
       - "5672:5672"
-      - "15672:15672"
\ No newline at end of file
+      - "15672:15672"
+  swift:
+    image: jeantil/openstack-keystone-swift:pike
```
[the remark this answers is not in the document]
Comment (chibenwa) [reply dockerfiles/run/docker-compose.yml]: Let's do it in another issue

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**